main.go

```diff
@@ -106,6 +106,15 @@ func details(c config.Config, name string) {
 }
 
 func add(c config.Config, path, name string) {
+
+
+	for _, c := range c.Commands {
+		if c.Name == name {
+			fmt.Println("Sorry, this ali is already taken.")
```

Comment:
change the text for something like `alias already in use`

Reply:
OK. I'll do it